### PR TITLE
Brian's updates

### PR DESCRIPTION
**Pull request by brivu**

### CODE REVIEW

**Comment by devinmarieb** [comment]
@felicianotech are these PRs by any chance duplicate work to what we are doing on docs? A lot of our PRs for image updates have been stuck in a review stage. I would hate for people to start doing work that's already being done. cc @brivu

**Comment by devinmarieb** [comment]
@brivu I know some of the new images won't work on version 2 with the new -browsers variant because it requires the browser orb, which in turn requires version 2.1. I'm not entirely sure which images this effects, but has that been taken into consideration here? I just want to double check as I see a lot of -browser updates. If we need to keep both versions for reference, we might need to create code tabs for server 2, server 3 and cloud.

File: jekyll/_cci2/language-php.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ version: 2
 jobs: # a collection of steps
   build: # runs not using Workflows must have a `build` job as entry point
     docker: # run the steps with Docker
-      - image: circleci/php:7.1-node-browsers # ...with this image as the primary container; this is where all `steps` will run
+      - image: cimg/php:8.0.14-node # ...with this image as the primary container; this is where all `steps` will run
```
Comment (felicianotech) [inline jekyll/_cci2/language-php.md:59]: This line below is no longer valid in next-gen images:

```
- run: sudo docker-php-ext-install zip
```

1. The zip extension is already installed. If we just wanted to show an example of how to install something else, we would use `pecl`.

File: jekyll/_cci2/language-php.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ version: 2
 jobs: # a collection of steps
   build: # runs not using Workflows must have a `build` job as entry point
     docker: # run the steps with Docker
-      - image: circleci/php:7.1-node-browsers # ...with this image as the primary container; this is where all `steps` will run
```
Comment (felicianotech) [inline jekyll/_cci2/language-php.md:59]: Looks like we dropped browsers. If this isn't a real project to run, then I think that's fine. If it is, then the browser tools orb and the browser variant are needed.

File: jekyll/_cci2/language-php.md [hunk 3]
```diff
@@ -123,7 +123,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/php:7.1-node-browsers
+      - image: cimg/php:8.0.14-node
```
Comment (felicianotech) [inline jekyll/_cci2/language-php.md:126]: Same comments below as I provided for above.

File: jekyll/_cci2/sample-config.md [hunk 4]
```diff
@@ -1000,11 +1000,11 @@ jobs:
 
   rake_test:
     docker:
-      - image: circleci/ruby:2.4-node-jessie
+      - image: circleci/ruby:3.1.0-node
         auth:
           username: mydockerhub-user
           password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
-      - image: circleci/postgres:9.4.12-alpine
+      - image: cimg/postgres:14.0.0
```
Comment (felicianotech) [inline jekyll/_cci2/sample-config.md:1007]: The tag `14.0.0` doesn't exist. It would be `14.0`.

File: jekyll/_cci2/sample-config.md [hunk 5]
```diff
@@ -1022,11 +1022,11 @@ jobs:
 
   precompile_assets:
     docker:
-      - image: circleci/ruby:2.4-node-jessie
+      - image: circleci/ruby:3.1.0-node
         auth:
           username: mydockerhub-user
           password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
-      - image: circleci/postgres:9.4.12-alpine
+      - image: cimg/postgres:14.0.0
```
Comment (felicianotech) [inline jekyll/_cci2/sample-config.md:1029]: Same as above.

**Review by felicianotech — CHANGES_REQUESTED** [review]

**Comment by Williamlliam** [comment]
@ThailandDistributionAgentoffice™



ในวันที่ ศ. 7 ม.ค. 2022 03:54 น. Brian Vu ***@***.***>
เขียนว่า:

File: jekyll/_cci2/language-php.md (hunk 2 above)
Comment (brivu) [reply jekyll/_cci2/language-php.md:59]: > If it is, then the browser tools orb and the browser variant are needed.

Do you mean I to drop the node variant and add the browsers variant with browser tool orbs?
Or did you mean I need to add the browsers image + orb in addition to the node variant?

File: jekyll/_cci2/language-php.md (hunk 1 above)
Comment (brivu) [reply jekyll/_cci2/language-php.md:59]: To clarify, do you mean `sudo docker-php-ext-install pecl` ?

**Pull request merged**